Tests/ImagineEngineTests/LabelTests.swift

```diff
@@ -122,4 +122,21 @@ final class LabelTests: XCTestCase {
         XCTAssertEqual(labelBClickCount, 1)
         XCTAssertEqual(clickedLabels, [labelB, labelA, labelA])
     }
+    
+    func testObservingRotate() {
+        let label = Label(text: "Greetings")
+
+        var labelRotateCount = 0
+        label.events.rotated.observe {
+            labelRotateCount += 1
+        }
+
+        XCTAssertEqual(labelRotateCount, 0)
+        XCTAssertEqual(label.rotation, 0)
+        label.rotation = 90
+        XCTAssertEqual(label.rotation, 90)
```

Comment:
I don't think you need to verify that the value of the property is the same that you just set, that makes the test harder to read IMO 🙂. Instead, I think you can focus on the relationship between changing `label.rotation` and `labelRotateCount`.

Reply:
Old habit to make sure that getter & setter works properly. :) I'll fix your findings and make another commit soon.